src/trixi_timeit.jl

```diff
@@ -0,0 +1,43 @@
+# Enable debug timings `@trixi_timeit timer() "name" stuff...`.
+# This allows us to disable timings completely by executing
+# `TimerOutputs.disable_debug_timings(TrixiBase)`
+# and to enable them again by executing
+# `TimerOutputs.enable_debug_timings(TrixiBase)`
+timeit_debug_enabled() = true
+disable_debug_timings() = TimerOutputs.disable_debug_timings(@__MODULE__)
+enable_debug_timings() = TimerOutputs.enable_debug_timings(@__MODULE__)
```

Comment:
It will still work in Trixi.jl if we import the stuff from here and call `Trixi.isable_debug_timings()`, correct?

Reply:
Yes, see https://github.com/trixi-framework/TrixiParticles.jl/pull/543, right at the beginning of the diff.